apps/package.json

```diff
@@ -14,6 +14,7 @@
     "test:unit": "grunt unitTest",
     "test:integration": "grunt integrationTest",
     "test:entry": "grunt karma:entry",
+    "test:interpreter": "./node_modules/@code-dot-org/js-interpreter-tyrant/bin/run.js --run --diff --root ./test/interpreter --interpreter ./test/interpreter/patched-interpreter.js --progress",
```

Comment:
The issue here is yarn 0.16.1. It doesn't actually put the `js-interpreter-tyrant` script in `node_modules/.bin` because `js-interpreter-tyrant` is under the `@code-dot-org` scope :(. So it ends up in `node_modules/@code-dot-org/.bin` which doesn't make any sense and isn't in the path. This has been fixed in newer versions of yarn. So we really need to upgrade.